app/views/home.php

```diff
@@ -1595,7 +1595,25 @@
                 <div class="swiper-container swiper-group-1 swiper-style-2 swiper">
                     <div class="swiper-wrapper pt-5">
                         <div class="swiper-slide">
-                            <div class="item-5 hover-up wow animate__animated animate__fadeIn">
+                            <?php  
+                                foreach($data['companies'] as $company){
+                                    echo 
+                                    '<div class="item-5 hover-up wow animate__animated animate__fadeIn">
```

Comment:
Không làm kiểu này, mà hãy đóng thẻ php lại rồi mới tới code html
Ví dụ:

```suggestion
                            <?php foreach($data['companies'] as $company): ?>
                                <div class="item-5 hover-up wow animate__animated animate__fadeIn">
```